serverless/messaging/api-cli/python-node-sns.mdx

```diff
@@ -89,7 +89,7 @@ for i := 0; i < 10; i++ {
 
 ### Create subscriptions to this topic (Go)
 
-#### Subscribe to an SQS queue from the same namespace
+#### Subscribe to an SQS queue from the same Scaleway Project
```

Comment:
Is this correct?

Reply:
No :-/

Subscription to SQS queues will not be possible anymore (consequence of the current namespace split).
Maybe we should disable this section or indicate that it's not possible yet.